client/add/box.ctrl.js

```diff
@@ -0,0 +1,22 @@
+const Constants = require('../../api/services/Constants');
+/**
+ * Controller for handling the dialog to add a new box
+ * @return {function} A controller that contains 2 test elements
+ */
+module.exports = function ($mdDialog) {
+  this.name = '';
+  this.description = '';
+  this.hide = function() {
+    $mdDialog.hide();
+  };
+  this.cancel = function() {
+    $mdDialog.cancel();
+  };
+  this.answer = function() {
+    $mdDialog.hide({
+      name: this.name,
+      description: this.description,
+      visibility: Constants.DEFAULT_BOX_VISIBILITY_SETTING
```

Comment:
Yeah. I put this because there were changed with the user format when I originally did this and I needed it before wiping my database for it not to error.